<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="HW7_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW1_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW2_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW6_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW3_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW4_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HW5_Slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW1_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW2_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW4_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW3_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW6_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW5_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW7_SpinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="MT1_Slider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="MT2_SpinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="MT2_Slider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="FINAL_Slider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="MT1_SpinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="Final_SpinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="MT1_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="FINAL_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Exam</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="MT2_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW1_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW2_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW3_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW4_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW5_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW6_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="HW7_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 7</string>
    </property>
   </widget>
   <widget class="QLabel" name="OFG_Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Final Grade:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Final_score_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>545</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>23</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>PIC 10B</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <item>
     <property name="text">
      <string>PIC 10B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC 10C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>721</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC 10B: 30% Homework + 20% Midterm 1 + 20% Midterm 2 + 30% Final Exam</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>831</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PIC 10C: 15% Homework + 30% Midterm + 55% Final Exam</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>HW1_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW1_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW2_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW2_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW3_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW3_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW4_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW4_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW5_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW5_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW6_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW6_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW7_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW7_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MT1_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MT1_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MT2_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MT2_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FINAL_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Final_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW1_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW1_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW2_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW2_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW3_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW3_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW4_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW4_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MT1_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MT1_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MT2_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MT2_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Final_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FINAL_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
